contracts/ERC721Factory.sol

```diff
@@ -75,7 +78,6 @@ contract ERC721Factory is Deployer, Ownable, ReentrancyGuard {
         address baseTokenAddress
     );
 
-
```

Comment:
Most changes in this PR are _not_ the goals of the issue but are formatting changes - spaces added, etc. It hurts the ability to review the real meat of the change for the goals of the issue. 

For the future, please create a separate issue & PR with something like "refactor to fix whitespace & other simple formatting". 

And ideally that's even done here. It will help to ensure that _this_ issue is well solved.

Reply:
I defer whether you keep the change in to whoever is reviewing the PR thoroughly. (As mentioned, I'm not.)

Reply:
Great!!